### PR TITLE
Debug description and config for Visual Studio Code

### PR DESCRIPTION
**Pull request by J03D03**
### Description

This PR adds a `launch.json` and a description to aid debugging `benchbuild` with Visual Studio Code.

----

### Example
- Set a breakpoint with `F9` in JustCompile for example at line 27 https://github.com/se-sic/VaRA-Tool-Suite/blob/7fe994fc282b68f24a4ff4dd9209fc94c1431df9/varats/varats/experiments/base/just_compile.py#L27
- Press `F5` to start debugging

### CODE REVIEW

File: docs/source/vara-ts/development.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@ First, take a look at our :ref:`install guide<Build VaRA with vara-buildsetup>`
 
 The :ref:`VaRA-TS API Reference` contains information about how to :ref:`work with the tool-suite<Tools>`, as well as how to add your own :ref:`research tools<Research Tool API>`, :ref:`experiments`, :ref:`reports`, :ref:`plots`, and more.
 
+:ref:`Debugging with Visual Studio Code` shows an example on how to debug benchbuild.
```
Comment (vulder) [inline docs/source/vara-ts/development.rst]:
```suggestion
:ref:`Debugging with Visual Studio Code` shows an example on how to debug benchbuild projects and experiments using the VSCode debugger.
```

File: docs/source/vara-ts/development.rst [hunk 2]
```diff
@@ -43,3 +45,12 @@ Module: test_utils
     :members:
     :undoc-members:
     :show-inheritance:
+
+
+-----
+
+Debugging with Visual Studio Code
+..............
```
Comment (vulder) [inline docs/source/vara-ts/development.rst]: The ... need to match the header length for sphinx to be happy :)

File: docs/source/vara-ts/development.rst [hunk 3]
```diff
@@ -43,3 +45,12 @@ Module: test_utils
...
+
+The .vscode/launch.json file in the VaRA-Tool-Suite repository contains a configuration for Visual Studio Code. With `F5` the given example is executed. It runs the command `benchbuild run -E JustCompile gzip`.
```
Comment (vulder) [inline docs/source/vara-ts/development.rst]:
```suggestion
The .vscode/launch.json file in the VaRA-Tool-Suite repository contains a configuration for Visual Studio Code.
With `F5` the given example is executed.
It runs the command `benchbuild run -E JustCompile gzip` by default, which can be adapted to debug other projects and experiments by changing the arguments that are passed to `benchbuild`.
```

File: docs/source/vara-ts/development.rst [hunk 4]
```diff
@@ -43,3 +45,12 @@ Module: test_utils
...
+To step through the JustCompile experiment a breakpoint has to be set in just_compile.py. `F9` can be used to set/unset a breakpoint at the current line.
```
Comment (vulder) [inline docs/source/vara-ts/development.rst]:
```suggestion
To step through the JustCompile experiment a breakpoint has to be set in just_compile.py.
`F9` can be used to set/unset a breakpoint at the current line.
```

**Review by vulder — APPROVED** [review]
I adapted  a bit of the wording, otherwise, this looks fine. Thanks for adding this config to make the debugging experience better ;)

File: docs/source/vara-ts/development.rst (hunk 2 above)
Comment (J03D03) [reply docs/source/vara-ts/development.rst]: Alright https://github.com/se-sic/VaRA-Tool-Suite/pull/462/commits/14b8784dc2d67fe61eeed1582c029634c3be132c :)

File: docs/source/vara-ts/development.rst (hunk 1 above)
Comment (J03D03) [reply docs/source/vara-ts/development.rst]: Thank you for the review

**Pull request merged**